app/models/invite_code.rb

```diff
@@ -0,0 +1,23 @@
+require 'coupon_code'
+
+class InviteCode < CouchRest::Model::Base
+  use_database 'invite_codes'
+  property :invite_code, String, :read_only => true
+  property :invite_count, Integer, :default => 0, :accessible => true
+
+  timestamps!
+
+  design do
+    view :by__id
```

Comment:
Not anymore - we briefly played with the idea to use the id as the invite code but it didn't seem practical (for starters, printed out and then copied down, it wasn't as user-friendly as the coupon code we generate now). 
Should we remove it?